install_config/cluster_metrics.adoc

```diff
@@ -162,71 +172,77 @@ to run cluster metrics with non-persistent data, running with non-persistent
 data does come with the risk of permanent data loss. However, metrics can still
 survive a container being restarted.
 
-In order to use non-persistent storage, you will need to set the
+In order to use non-persistent storage, you must set the
 `*USE_PERSISTENT_STORAGE*`
-link:../install_config/cluster_metrics.html#creating-the-deployer-template[template
-option] to `false` for the *metrics deployer*.
+link:../install_config/cluster_metrics.html#modifying-the-deployer-template[template
+option] to `false` for the Metrics Deployer.
 
 [[metrics-deployer]]
-
 == Metrics Deployer
 
-The *metrics deployer* deploys and configures all of the metrics components. You
+The Metrics Deployer deploys and configures all of the metrics components. You
 can configure it by passing in information from
-link:../dev_guide/secrets.html[secrets] and by passing parameters to the
-*metrics deployer*'s link:../dev_guide/templates.html[template].
+link:../dev_guide/secrets.html[secrets] and by passing parameters to the Metrics
+Deployer's link:../architecture/core_concepts/templates.html[template].
 
-=== Metrics Deployer Secrets
+[[metrics-deployer-using-secrets]]
+=== Using Secrets
 
-By default, the *metrics deployer* auto-generates self-signed certificates for
-use between components. Since these are self-signed certificates they will not
-be automatically trusted by a web browser and it is advisable to use your own
-certificates for anything being accessed outside of the OpenShift cluster. This
-is especially important for the Hawkular Metrics server as it needs to be accessible
-in a browser for the console to function.
+By default, the Metrics Deployer auto-generates self-signed certificates for
+use between components. Because these are self-signed certificates, they are not
+automatically trusted by a web browser; therefore, it is advisable to use your
+own certificates for anything being accessed outside of the OpenShift cluster.
+This is especially important for the Hawkular Metrics server as it must be
+accessible in a browser for the web console to function.
 
-To provide your own certificates and replace the internally used ones, you
-can pass these values as link:../dev_guide/secrets.html[secrets] to the
-*metrics deployer*.
+The Metrics Deployer requires that you manually create a *metrics-deployer*
+secret whether you are link:#metrics-using-secrets-byo-certs[providing your own
+certificates] or link:#metrics-using-secrets-autogenerated[using generated
+self-signed certificates].
```

Comment:
Tried to make this issue more obvious, and split the following information into 2 subsections.